src/HuC1.md

```diff
@@ -1,51 +1,53 @@
 # HuC1
 
-HuC1 is an MBC used by some Game Boy games which besides doing the usual
-MBC stuff, also provides IR comms. A lot of sources on the internet said
-that HuC1 was \"similar to MBC1\", but they didn\'t provide any detail.
-I took a look, and it turns out that HuC1 differs from MBC1 quite a lot.
+HuC1 is an MBC developed by Hudson Soft. It implements ROM and RAM
+banking, and also provides infrared communication. Many sources on the
+internet said that HuC1 was “similar to MBC1”, without providing any
```

Comment:
I would replace this with (starting with "Many sources on the", from the previous line)

```
Unlike generally believed, the HuC1 differs from MBC1 significantly.
```

Reply:
But that reads poorly in English – if you really want to rework it in that direction, it should be “Contrary to popular belief…”  But then you’re not explaining where the popular belief came from (i.e. these repeated claims on various web sites).

Anyway, if you want to endlessly argue about wording like this, count me out.  I have more important things to do.  I just wanted to get write up my findings on the HuC-3 while it was fresh in my mind after doing reverse-engineering and emulation work on it.

Reply:
I don't think the "many sources on the Internet" part belongs in Pan Docs at all, which would solve the problem.

> Anyway, if you want to endlessly argue about wording like this, count me out. I have more important things to do. I just wanted to get write up my findings on the HuC-3 while it was fresh in my mind after doing reverse-engineering and emulation work on it.

Considering that all of the technical tidbits have been approved by both of us, I'd entirely understand if you decline to participate in the grammar/presentation bikeshedding; we can make phrasing changes ourselves. Regardless, thank you *very much* for your research, and for caring enough to reshare it. Know that it's appreciated.

Reply:
> I don't think the "many sources on the Internet" part belongs in Pan Docs at all, which would solve the problem.

Well, it gives historical context for JRRA’s comments, and it’s a sad fact that there are still many outdated websites that misleadingly say the HuC1 and HuC-3 are “similar to MBC1” or “similar to MBC5”.  It’s another thing carried over from the original blog post.